README.md

```diff
@@ -3,36 +3,55 @@
 [![Docker Build Status](https://img.shields.io/docker/build/esmvalgroup/esmvaltool.svg)](https://hub.docker.com/r/esmvalgroup/esmvaltool/)
 [![Anaconda-Server Badge](https://anaconda.org/esmvalgroup/esmvalcore/badges/installer/conda.svg)](https://conda.anaconda.org/esmvalgroup)
 
 
-ESMValTool Core: core functionalities for the ESMValTool, a community diagnostic and performance metrics tool for routine evaluation of Earth system models in CMIP
+ESMValTool Core: core functionalities for the ESMValTool, a community
+diagnostic and performance metrics tool for routine evaluation of Earth System
+Models in CMIP.
 
 # Getting started
-This is the development branch for ESMValTool Core. ESMValTool version 2 is under rapid development, an installation from source is recommended at the moment.
-
-## Installing from source [recommended]
-Please see [CONTRIBUTING.md](https://github.com/ESMValGroup/ESMValCore/blob/development/CONTRIBUTING.md) for instructions on installing ESMValTool from source.
 
 ## Installing from Anaconda
 The Anaconda package can be found on [ESMValGroup Anaconda Channel.](https://anaconda.org/ESMValGroup)
 
-If you already installed Anaconda, you can install ESMValTool by running:
+If you already have
+[Anaconda installed](https://conda.io/projects/conda/en/latest/user-guide/install/index.html),
+you can install ESMValTool Core by running:
 ```
 conda install -c esmvalgroup -c conda-forge esmvalcore
 ```
 
+## Installing from source
+If you need the very latest features, please see
+[CONTRIBUTING.md](https://github.com/ESMValGroup/ESMValCore/blob/development/CONTRIBUTING.md)
+for instructions on installing ESMValTool Core from source.
+
+## Using ESMValTool Core as a Python library
+The ESMValTool Core package provides various functions for:
+- Finding data in a directory structure typically used for CMIP data
+- Reading CMOR tables and using those to check model and observational data.
+- ESMValTool preprocessor functions for e.g. regridding, vertical
+  interpolation, statistics, correcting (meta)data errors, etcetera.
+
 ## Running ESMValTool
-- Review `config-user.yml`. To customize for your system, create a copy, edit and use the command line option `-c` to instruct `esmvaltool` to use your custom configuration.
-- Optionally install the ESMValTool recipes and diagnostics
-- Run e.g. `esmvaltool -c ~/config-user.yml examples/recipe_python.yml
+- Review `config-user.yml`. To customize for your system, create a copy, edit
+  and use the command line option `-c` to instruct `esmvaltool` to use your
+  custom configuration file.
+- Install the [ESMValTool](https://github.com/ESMValGroup/ESMValTool)
+  to run [ESMValTool recipes and diagnostics](https://esmvaltool.readthedocs.io/en/latest/recipes/index.html)
+- Run e.g. `esmvaltool -c ~/config-user.yml examples/recipe_python.yml` after
```

Comment:
```suggestion
- Run e.g. `esmvaltool -c ~/config-user.yml recipes/examples/recipe_python.yml` after
```

Reply:
There is no need to add `recipes/`, in fact that will only work if you're in a certain location of the filesystem. `examples/recipe_python.yml` will work regardless of the current working directory